lib/auth/tls_test.go

```diff
@@ -45,13 +47,35 @@ import (
 	"github.com/gravitational/teleport/lib/session"
 	"github.com/gravitational/teleport/lib/tlsca"
 	"github.com/gravitational/teleport/lib/utils"
+	"github.com/gravitational/trace/trail"
 
 	"github.com/gravitational/trace"
 	"github.com/jonboulle/clockwork"
 	"github.com/pquerna/otp/totp"
 	"gopkg.in/check.v1"
 )
 
+// MockAuthServiceClient stubs grpc AuthServiceClient interface.
+type mockAuthServiceClient struct {
+	proto.AuthServiceClient
+	errType error
```

Comment:
nit: how about `err` or `deleteUserErr`?

Reply:
revised to `err`